configs/minkunet/minkunet_w16_8xb2-15e_semantickitti.py

```diff
@@ -0,0 +1,10 @@
+_base_ = ['./minkunet_w32_8xb2-15e_semantickitti.py']
+
+model = dict(
+    backbone=dict(
```

Comment:
Please add README.md and describe the meaning of this config name and the wave of precision.